docs/dev/development.md

```diff
@@ -1,70 +1,90 @@
+
 ## IDEA
 
-Cloudbreak can be imported into IDEA as gradle project. Once it is done, you need to import the proper code formatter by using the __File -> Import Settings...__ menu and selecting the `idea_settings.jar` located in the `config` directory in Cloudbreak git repository.
+Configure -> Project Defaults -> Project Structure _> Set SDK to your Java version
+
+Cloudbreak can be imported into IDEA as gradle project. Once it is done, you need to import the proper code formatter by using the __File -> Import Settings...__ menu and selecting the `idea_settings.jar` located in the `config/idea` directory in Cloudbreak git repository.
 
 To launch the Cloudbreak application execute the `com.sequenceiq.cloudbreak.CloudbreakApplication` class with VM options:
 ```
 -XX:MaxPermSize=1024m
 -Dcb.cert.dir=FULL_PATH_OF_THE_CERTS_DIR_GENERATED_BY_CBD
 -Dcb.client.id=cloudbreak
--Dcb.client.secret=cbsecret2015
--Dcb.db.port.5432.tcp.addr=192.168.59.103
+-Dcb.client.secret=CB_SECRET_GENERATED_BY_CBD
+-Dcb.db.port.5432.tcp.addr=192.168.99.100
 -Dcb.db.port.5432.tcp.port=5432
--Dcb.identity.server.url=http://192.168.59.103:8089
+-Dcb.identity.server.url=http://192.168.99.100:8089
 -Dserver.port=9091
 -Dcb.schema.migration.auto=true
 ```
 
-The `-Dcb.cert.dir=FULL_PATH_OF_THE_CERTS_DIR_GENERATED_BY_CBD` value above needs to be replaced with the full path of `certs` directory generated by Cloudbreak Deployer e.g. `-Dcb.cert.dir=/Users/myusername/prj/cbd-local/certs`.
+The `-Dcb.cert.dir=FULL_PATH_OF_THE_CERTS_DIR_GENERATED_BY_CBD` value above needs to be replaced with the full path of `certs` directory generated by Cloudbreak Deployer e.g. `/Users/YOUR_USERNAME/YOUR_PROJECT_DIR/cbd-local`.
+
+The `-Dcb.client.secret=CB_SECRET_GENERATED_BY_CBD` value has to be replaced with the value of UAA_DEFAULT_SECRET from the cdb-local/Profile file.
 The databse migration is ran automatically by Cloudbreak, this automated migration could be turn off through the `-Dcb.schema.migration.auto=false` VM option.
 
 ## Command line
 
-To run Cloudbreak from command line you have to create a property file, for example application.properties, with the content below, and execute `./gradlew bootRun -Dspring.config.location=file:/path/of/property/application.properties` command at project root.
+To run Cloudbreak from command line you have to list the JVM parameters from above for gradle:
+
 ```
--Dcb.cert.dir=FULL_PATH_OF_THE_CERTS_DIR_GENERATED_BY_CBD
--Dcb.client.id=cloudbreak
--Dcb.client.secret=cbsecret2015
--Dcb.db.port.5432.tcp.addr=192.168.59.103
--Dcb.db.port.5432.tcp.port=5432
--Dcb.identity.server.url=http://192.168.59.103:8089
--Dserver.port=9091
+./gradlew :core:bootRun -PjvmArgs="-Dcb.cert.dir=FULL_PATH_OF_THE_CERTS_DIR_GENERATED_BY_CBD \
+-Dcb.client.id=cloudbreak \
+-Dcb.client.secret=CB_SECRET_GENERATED_BY_CBD \
+-Dcb.db.port.5432.tcp.addr=192.168.99.100 \
+-Dcb.db.port.5432.tcp.port=5432 \
+-Dcb.identity.server.url=http://192.168.99.100:8089 \
```

Comment:
the IP of the docker machine instance called `cbd`, it replacable with `$(docker-machine ip cbd)`